src/main/scala/core/screenings/ScreeningService.scala

```diff
@@ -0,0 +1,67 @@
+package core.screenings
+
+import java.sql.Timestamp
```

Comment:
Didn't check documentation good enough to notice it has support since 3.3.0, it would be simpler to have java.time LocalDateTime. It will probably map into timestamp in database, but it's shorter and simpler String mapping would make it easier to make requests because of T instead of <space> as spacer.